Tests doesn't pass on go1.10

Hi guys, looks like tests doesn't pass after go1.10 update:

```cp vendor/mruby/build/host/lib/libmruby.a .
Checking code with gofmt..
gofmt -s *.go >/dev/null
sh golint.sh
Installing golint into your GOPATH...
Checking with golint...
go test -v
# github.com/DamnWidget/aim/vendor/github.com/mitchellh/go-mruby
./array_test.go:38: Fatalf format %s has arg valType of wrong type mruby.ValueType
./hash_test.go:33: Fatalf format %s has arg valType of wrong type mruby.ValueType
./hash_test.go:58: Fatalf format %s has arg value.Type() of wrong type mruby.ValueType
./parser_test.go:19: Fatalf format %s has arg warns of wrong type []*mruby.ParserMessage
./parser_test.go:44: Fatalf format %s has arg warns of wrong type []*mruby.ParserMessage
./value_test.go:245: Fatalf format %s has arg cType of wrong type mruby.ValueType
FAIL	github.com/DamnWidget/aim/vendor/github.com/mitchellh/go-mruby [build failed]
make: *** [Makefile:27: test] Error 2```

--- a/array_test.go
+++ b/array_test.go
@@ -35,7 +35,7 @@ func TestArray(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 	if valType := value.Type(); valType != TypeFalse {
-		t.Fatalf("bad type: %s", valType)
+		t.Fatalf("bad type: %v", valType)
 	}
 	if value.String() != "false" {
 		t.Fatalf("bad: %s", value)

--- a/hash_test.go
+++ b/hash_test.go
@@ -30,7 +30,7 @@ func TestHash(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 	if valType := value.Type(); valType != TypeFalse {
-		t.Fatalf("bad type: %s", valType)
+		t.Fatalf("bad type: %v", valType)
 	}
 	if value.String() != "false" {
 		t.Fatalf("bad: %s", value)
@@ -55,7 +55,7 @@ func TestHash(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 	if value.Type() != TypeArray {
-		t.Fatalf("bad: %s", value.Type())
+		t.Fatalf("bad: %v", value.Type())
 	}
 	if value.String() != `["foo", "baz"]` {
 		t.Fatalf("bad: %s", value)

--- a/parser_test.go
+++ b/parser_test.go
@@ -16,7 +16,7 @@ func TestParserGenerateCode(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 	if warns != nil {
-		t.Fatalf("warnings: %s", warns)
+		t.Fatalf("warnings: %v", warns)
 	}
 
 	proc := p.GenerateCode()
@@ -41,7 +41,7 @@ func TestParserParse(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 	if warns != nil {
-		t.Fatalf("warnings: %s", warns)
+		t.Fatalf("warnings: %v", warns)
 	}
 }
 

--- a/value_test.go
+++ b/value_test.go
@@ -242,7 +242,7 @@ func TestMrbValueType(t *testing.T) {
 			t.Fatalf("loadstring failed for case %#v: %s", c, err)
 		}
 		if cType := r.Type(); cType != c.Expected {
-			t.Fatalf("bad type: got %s, expected %s", cType, c.Expected)
+			t.Fatalf("bad type: got %v, expected %v", cType, c.Expected)
 		}
 	}
 }

